project/legislation/openstates/pyopenstates.py

```diff
@@ -0,0 +1,450 @@
+# -*- coding: utf-8 -*-
```

Comment:
Rather than storing this library in our Git repository/history, see if it can be included as a dependency and installed via pip. See the [pyopenstates documentation](https://github.com/openstates/pyopenstates#installation) for installation instructions via pip.